### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by twling**
Add info for deployment of the Ploigos reference application, reformatted and updated the abstract information.

### CODE REVIEW

File: docs/modules/workflow/con-ploigos-workflow-infrastructure.adoc [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
+[cols="a,a,a,a,a,a",options="header"]
+|===
+| Node
+| CPUs
+| Memory (GB)
+| Disk (GB)
+| AWS EC2 Instance Type
+| Sizing Source
+
+| Master 0
```
Comment (itewk) [inline docs/modules/workflow/con-ploigos-workflow-infrastructure.adoc]: @twling change `master` to `control`

**Review by itewk — CHANGES_REQUESTED** [review]

**Comment by itewk** [comment]
@jflowers @andykrohg this is mostly an update to add instructions for installing the operator. can you all take a look?

you can see rednered version here: https://twling.github.io/ploigos-docs/

File: docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc [hunk 2]
```diff
@@ -2,4 +2,175 @@
...
+EOF
+----
++
+. Create a project named *devsecops* for the pipeline tooling+
++
+----
+oc new-project devsecops
+----
++
+. (Optional) Remove too-restrictive limit ranges *NOTE: Required for RHPDS clusters*
```
Comment (andykrohg) [inline docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc]: This won't break if the`LimitRange` doesn't exist - we should probably just make this step required

File: docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc [hunk 3]
```diff
@@ -2,4 +2,175 @@
...
++
+----
+oc delete limitrange --all -n devsecops
+----
++
+. Install the 'Ploigos Software Factory Operator'
+* In the OCP Web Console, navigate to:
+** *Operators -> OperatorHub*
+** Search for "Ploigos Software Factor Operator"
```
Comment (andykrohg) [inline docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc]: Factory*

File: docs/modules/workflow/con-ploigos-workflow-infrastructure.adoc (hunk 1 above)
Comment (twling) [reply docs/modules/workflow/con-ploigos-workflow-infrastructure.adoc]: Easy enough to change, but would this be confusing until openshift changes their terminology?

File: docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc (hunk 2 above)
Comment (twling) [reply docs/modules/workflow/proc-ploigos-workflow-reference-impl.adoc]: I'll change this, but we should collect some info on what valid limits are for TSSC. Unlimited seems too open.

**Review by itewk — APPROVED** [review]

**Pull request merged**